### PR TITLE
feat: add go 1.22

### PR DESCRIPTION
**Pull request by vbehar**
add both alpine and debian-based variants

### CODE REVIEW

**Comment by MrEsL** [comment]
any update on this ?

**Comment by vbehar** [comment]
FYI I've pushed the 1.22 on https://github.com/vbehar/devspace-containers/pkgs/container/devspace-containers%2Fgo 
from this branch: https://github.com/vbehar/devspace-containers/tree/go

**Comment by MrEsL** [comment]
Thanks for your comment, I basically did the same exact thing before you responded :)

Would still love to use the image from the official devspace-containers

File: .github/workflows/go.yaml [hunk 1]
```diff
@@ -25,6 +25,8 @@ jobs:
       fail-fast: false
       matrix:
         include:
+          - tag: "1.22"
```
Comment (ThomasK33) [inline .github/workflows/go.yaml]: We don't want to have non-alpine tags for now. Would you mind removing that one from the matrix, please?

**Review by ThomasK33 — CHANGES_REQUESTED** [review]
The PR is looking good overall. 

Could you also remove the Debian 1.22 image from the matrix, please?

File: .github/workflows/go.yaml (hunk 1 above)
Comment (vbehar) [reply .github/workflows/go.yaml]: ok done!

**Review by ThomasK33 — APPROVED** [review]
Thank you very much.

**Pull request merged**